server/lib/src/server/access/mod.rs

```diff
@@ -626,11 +622,7 @@ pub trait AccessControlsTransaction<'a> {
                         false
                     } else if !requested_rem.is_subset(&rem) {
                         security_error!("requested_rem is not a subset of allowed");
-                        security_error!(
-                            "requested_rem: {:?} !⊆ allowed: {:?}",
-                            requested_rem,
-                            rem
-                        );
+                        security_error!("requested_rem: {:?} !⊆ allowed: {:?}", requested_rem, rem);
```

Comment:
What version of rust are you on? Looks like cargo fmt fights here.

Reply:
rustc 1.77.0-nightly (5518eaa94 2024-01-29)